ddtrace/contrib/botocore/patch.py

```diff
@@ -27,99 +27,92 @@
 # Original botocore client class
 _Botocore_client = botocore.client.BaseClient
 
-ARGS_NAME = ('action', 'params', 'path', 'verb')
-TRACED_ARGS = ['params', 'path', 'verb']
+ARGS_NAME = ("action", "params", "path", "verb")
```

Comment:
It looks like you ran black on this but did not remove it from the exclude list in `pyproject.yaml`, right?

Reply:
Now that you point that out I have realised that there are exclusions in `pyproject.toml`. I have black set to run on save in VS Code. Is there a reason why `botocore` is excluded from black?

Reply:
We haven't gotten to formatting it yet. We went for a deny list approach to ensure all new files added are formatted, and we can slowly update all files to eventually be formatted.

If you want, you can submit a separate PR for formatting `botocore` and removing it from `pyproject.toml` exclusion.